trusted-committer-article.md

```diff
@@ -0,0 +1,232 @@
+
+## Introduction
+
+- tbd
+
+## Introducing The Trusted Committer Role
+
+The Trusted Committer (TC) role is one of the key roles in an InnerSource
+community. It is both a challenging and a rewarding role to fulfill and it is
+instrumental for the success of any InnerSource community. 
+
+Generally speaking, the TC role is defined by its responsibilities, rather than
+by its privileges. On a very high level, a TC represents the interests of both
+their InnerSource community and the products the community is building. So as
+a TC, you'll have both tech oriented and community oriented responsibilities
+and we'll explore both of these dimensions in the following sections.
+
+Before we go into the details of what a TC actually does, let's spend some time
+contrasting the TC role to other roles in IS and why we think the name is both
+apt and important.
+
+- Contributor
+    - makes contributions
+    - sometimes part of the community
+    - sometimes sent by another part of the organization to develop a feature
+      for it. This is why contributors are sometimes referred to as Guests. 
+    - responsible for "fitting in"/conforming to the host team and it's 
+      expectations and processes
+    - similar to an Apache Developer or committer
+
+- Trusted Committer
+    - Sometimes referred to as a host.
+    - In this analogy, he is both resonsible for building the house and setting
+      the house rules to make sure his guests are comfortable.
+    - In comparison to the Contributor, a TC has rights to push code closer to
+      production
+- Product Owner
+    - Responsible for defining and prioritizing requirements and stories for
+      the community to implement.
+    - Also interacts directly and frequently with the trusted committers
+    - Distinction between the guest and host PO - have to negotiate
+      requirements, if necessary.
+    - Depending on the type of InnerSource community, there might not be a PO
+      from the outset. Especially true in case of innovation oriented and
+      bottom-up driven communities. POs come into play when a productization
+      of an InnerSource communities products becomes relevant.
+
+    ? In the Guest/Host analogy, what would be the analogon of a PO?
+
+## Why role names matter
+
+- The role of the TC is present in every successful InnerSource community but
+  not every community uses that name.
+- True for myself - I used the term Maintainer. It turns out that conficts with
+  the technical role "Maintainer" defined by GitHub, for instance. 
+- Apache e. g. also uses the term Committer, but attaches fewer and mostly
+  tech oriented responsibilities to that role. The TC role with its additional
+  community oriented responsibilities goes beyond that.
+- Similar to how naming is important in writing software, it is, too, for the
+  roles in an InnerSource project so that everyone has the same notion of 
+  the role.
+- The "Trusted" in TC conveys that she is trusted by both her community and
```

Comment:
Also in many ways a TC builds "Trust" for the project, by being open and transparent.

Reply:
They also build the *community* with their work. Your point about their continuing influence is important I think, and will feed a virtuous cycle: Good mentoring on pull requests -> higher levels of trust from contributors -> more people being willing to contribute -> more pull requests.

Reply:
Excellent points, @mrsanz and @cewilliams . Will update the doc accordingly.